hardware/raspberrypi/bootmodes/usb.md

```diff
@@ -7,7 +7,7 @@ There are two separate boot modes for USB:
   * [USB mass storage boot](msd.md)
   * [Network boot](net.md)
 
-Note that network boot is only possible on Raspberry Pi models that have a built-in wired Ethernet interface.
+**USB host boot is available on the Raspberry Pi 3B, 3B+, 3A+ and Raspberry Pi 2B v1.2 models only.**
```

Comment:
Not sure that this needs to be mentioned here, given that it's also mentioned in host.md? :man_shrugging:

Reply:
It's already mentioned on this page, I'm just changing the wording.

Reply:
I personally think it should be mentioned on both pages, but I'm happy to change it. Possibly one for the copy editors since it is kind of style related.

Reply:
It's just that this page mentions both "USB device boot" and "USB host boot" so I wondered why you were specifically only adding a note about "USB host boot"? :man_shrugging:  (and I like to avoid duplication where possible, but that might just be a personal thing)

I agree it's probably a copy-editors' decision.

Reply:
I agree that we probably don't need the warning about which Pi's do USB host boot in that page. I have removed that, and instead added '(only available on certain models)' to the end of the first sentence. I think that is simpler, but still retains a warning that all Pi's don't do the USB boot modes. If folk want to know more, it should be obvious that they just click on the relevant sub-page.